rfc/0084-remove-0081-active-user-beacon.md

```diff
@@ -0,0 +1,42 @@
+# Revert RFC 81: Active User Beacon
+
+## Summary
+
+This RFC reverts the RFC 81: active user beacon.
+
+## Motivation
+
+A significant number of developers and users have expressed unhappiness at the current opt-out behaviour of the active user beacon, and want it changed or removed. Since not annoying users is important, I'd like to remove the beacon entirely.
+
+Some people have made the argument that other projects have similar amounts (or significantly more) user tracking. I don't think this justifies this inclusion - each case of tracking should be looked at separately.
```

Comment:
In my opinion this whole document is unfit to document a feature removal (couldn't we just remove the previous RFC and use the GitHub conversation as a summary ?), but if we were to keep the "RFC deletion RFC" I think this section should be rewritten to more accurately describe the various arguments being made.